src/modules/auth/controllers/verifyOtp.ts

```diff
@@ -0,0 +1,52 @@
+import { Request, Response } from 'express';
```

Comment:
Ok, is it that u added the respective utilities to it's controller file in this place

Reply:
Having fixed that, just fixed the build error, so as to deploy the it